tests/test_common.py

```diff
@@ -40,34 +45,36 @@ def test_FoundModule():
     ["stmt", "result"],
     [
         ("import ast", ["ast"]),
-        ("import ast, sys", ["ast", "sys"]),
-        ("from sys import version", ["sys"]),
-        ("from os import path", ["os"]),
-        ("import distutils.command.check", ["distutils"]),
+        ("import ast, pathlib", ["ast", "pathlib"]),
+        ("from pathlib import Path", ["pathlib"]),
+        ("from string import hexdigits", ["string"]),
+        ("import distutils.command.check", ["distutils.command.check"]),
```

Comment:
This last line is a real change in the test. I don't understand it.

The test was broken before as it was ignoring the `result` input.